script/build.js

```diff
@@ -126,8 +126,8 @@ const ignore = require('metalsmith-ignore');
 const ignoreList = [];
 if (options.buildtype === 'production') {
   ignoreList.push('education/gi-bill/post-9-11/status.md');
-  ignoreList.push('pensions/application.md');
-  ignoreList.push('burials-and-memorials/application.md');
+  ignoreList.push('pension/application/527.md');
+  ignoreList.push('burials-and-memorials/application/530.md');
```

Comment:
Why is `pension` singular but `burials and memorials` plural?

Not sure who to ask, so I'll flag @J-Quag (who can then flag whomever is the person to ask).